spec/game_runner_spec.rb

```diff
@@ -20,5 +20,25 @@
 
       expect(stdout.string).to eq(grid_template)
```

Comment:
I'm thinking I'll need to change this to `expect(stdout.string).to include(grid_template)`